android/assets/jsons/translations/English.properties

```diff
@@ -1252,6 +1252,23 @@ You are not allowed to spectate! =
  # Requires translation!
 Couldn't download the latest game state! = 
 
+## Chat
+Chat = 
+UncivChat™ = 
+System = 
+Server = 
+Unknown = 
+one time = 
+Welcome to Chat! = 
+Successfully connected to WebSocket server! = 
+Successfully re-established WebSocket connection! = 
```

Comment:
We can just use the previous connection message, no need for 2 messages

Reply:
Well this is a mistake that I was also thinking about. We probably need a connection closed message and connection established message. But for some unknown reason i did not add connection closed message and instead added re-established message. Not sure what I was thinking at that time.

Reply:
fixed by https://github.com/yairm210/Unciv/pull/13599/commits/e81f1dd320c742cd0a64fad4fd5a054a2f60baed

Reply:
Ok, so I already had a global message for close. I think I remember now. I wanted a different message for re-connection. Anyways, fixed by https://github.com/yairm210/Unciv/pull/13599/commits/35c3ca886dfabd191f35bb8195b0fc0f0aaef0f3.